documentation/publish/action.md

```diff
@@ -60,4 +60,5 @@ env:
 
 :::tip
 If you maintain an integration, you can also validate your integration with [hassfest](https://developers.home-assistant.io/blog/2020/04/16/hassfest).
+For both plugins and cards, an easy way to automatically validate and format it would be with [ha-blueprint](https://github.com/KTibow/ha-blueprint).
```

Comment:
`FORMAT_CODE: DISABLED` has been around for a while. I'm not comfortable defaulting it to false; that would be a breaking change in the repos that already use it.
```suggestion
If you want a more combined GitHub Action, you could use [ha-blueprint](https://github.com/KTibow/ha-blueprint) (don't forget to set `FORMAT_CODE: DISABLED` if you don't want your commits to be amended with formatting).
```